# [PATCH] Only show template start/end comments for super admins
# Author: yakov

--- a/inc/class_templates.php
+++ b/inc/class_templates.php
@@ -106,7 +106,7 @@ class templates
 
 		if($htmlcomments && $template !== false)
 		{
-			if($mybb->settings['tplhtmlcomments'] == 1)
+			if($mybb->settings['tplhtmlcomments'] == 1 && $mybb->usergroup['issupermod'] == 1)
 			{
 				$template = "<!-- start: ".htmlspecialchars_uni($title)." -->\n{$template}\n<!-- end: ".htmlspecialchars_uni($title)." -->";
 			}